contrib/static-opus/Makefile

```diff
@@ -0,0 +1,182 @@
+
+$(OGG_ARCHIVE):
+	$(CURL) "$(OGG_URL)" -o "$(OGG_ARCHIVE)"
+
+ogg/autogen.sh: $(OGG_ARCHIVE)
+	@test -f $@ \
+	|| ( mkdir -p ogg \
+	&& $(EXTRACT) $(OGG_ARCHIVE) -C ogg )
+
+ogg/configure: ogg/autogen.sh
+	cd ogg \
+	&& mkdir -p m4 \
+	&& ./autogen.sh
+
+ogg/Makefile: ogg/configure
+	cd ogg \
+	&& ./configure --disable-shared
+
+lib/libogg.a: ogg/Makefile
+	cd ogg \
+	&& $(MAKE) -j$(THREADS) \
+	&& mkdir -p ../include/ogg ../lib \
+	&& cp -v include/ogg/*.h ../include/ogg \
+	&& cp -v src/.libs/libogg.a ../lib
+
+ogg/clean:
+	cd ogg \
+	&& $(MAKE) clean \
+	&& rm -f configure Makefile
+
+##
+#  Opus Library
+#  ============
+opus: lib/libopus.a
+
+$(OPUS_ARCHIVE):
+	$(CURL) "$(OPUS_URL)" -o "$(OPUS_ARCHIVE)"
+
+opus/autogen.sh: $(OPUS_ARCHIVE)
+	@test -f $@ \
+	|| ( mkdir -p opus \
+	&& $(EXTRACT) $(OPUS_ARCHIVE) -C opus )
+
+opus/configure: opus/autogen.sh
+	cd opus \
+	&& mkdir -p m4 \
+	&& ./autogen.sh
+
+opus/Makefile: opus/configure
+	cd opus \
+	&& ./configure --disable-shared --disable-extra-programs --disable-doc
+
+lib/libopus.a: opus/Makefile
+	cd opus \
+	&& $(MAKE) -j$(THREADS) \
+	&& mkdir -p ../include/opus ../lib \
+	&& cp -v include/*.h ../include/opus \
+	&& cp -v .libs/libopus.a ../lib
+
+opus/clean:
+	cd opus \
+	&& $(MAKE) clean \
+	&& rm -f configure Makefile
+
+##
+#  Opusfile Library
+#  ================
+opusfile: opusfile-message
+
+pkg.m4:
+	$(CURL) "$(OPUSFILE_PKG_URL)" -o "pkg.m4"
+
+$(OPUSFILE_ARCHIVE):
+	$(CURL) "$(OPUSFILE_URL)" -o "$(OPUSFILE_ARCHIVE)"
+
+opusfile/autogen.sh: $(OPUSFILE_ARCHIVE)
+	@test -f $@ \
+	|| ( mkdir -p opusfile \
+	&& $(EXTRACT) $(OPUSFILE_ARCHIVE) -C opusfile )
+
+opusfile/configure: opusfile/autogen.sh pkg.m4
+	cd opusfile \
+	&& mkdir -p m4 \
+	&& cp ../pkg.m4 m4/ \
+	&& ./autogen.sh
+
+opusfile/Makefile: opusfile/configure lib/libopus.a lib/libogg.a
+	cd opusfile && \
+	DEPS_LIBS="-L../lib -lopus -logg" \
+	DEPS_CFLAGS="-I../include -I../include/opus -I../include/ogg" \
+	./configure --disable-shared --disable-doc --disable-http --disable-examples
+
+# We run make twice below. We hide the output of the first run because the
+# auto-tools generated Makefile launches sed and libtool without quoting the
+# current working directory, so those fail when run inside a directory structure
+# containing one or more spaces.
+#
+# Regardless of the sed/libtool issue, if all went well the resulting library will
+# exist, which is why we run make a second time. The second make pass doesn't
+# rerun the sed and libtool comands, so it acts as a safety check. If there's a
+# actual build problem with the source code then it will be caught in this
+# second make run and fail without producing a library.
+#
+lib/libopusfile.a: opusfile/Makefile
+	cd opusfile \
+	&& ( $(MAKE) -j$(THREADS) > make.log 2>&1 || $(MAKE) ) \
+	&& mkdir -p ../include/ ../lib \
+	&& cp -v include/*.h ../include/opus \
+	&& cp -v .libs/*.a ../lib
+
+.PHONY: opusfile-message
+opusfile-message: lib/libopusfile.a
+	@echo "\n"\
```

Comment:
`echo -e`

Also, I suggest adding a single space before `\` indicating line-break, just for readability.

Reply:
Found a way to `define` a text block and print it with new lines using the $info( ... ) directive; thank you stack exchange!